hack/verify-shellcheck.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# set -o errexit
+set -o nounset
+set -o pipefail
+
+OS="unknown"
+if [[ "${OSTYPE}" == "linux"* ]]; then
+  OS="linux"
+elif [[ "${OSTYPE}" == "darwin"* ]]; then
+  OS="darwin"
+fi
+
+# shellcheck source=./hack/utils.sh
+source "$(dirname "$0")/utils.sh"
+ROOT_PATH=$(get_root_path)
+
+# create a temporary directory
+TMP_DIR=$(mktemp -d)
+
+# cleanup on exit
+cleanup() {
+  echo "Cleaning up..."
+  rm -rf "${TMP_DIR}"
+}
+trap cleanup EXIT
+
+# install shellcheck (x64 only!)
+cd "${TMP_DIR}" || exit
+VERSION="shellcheck-v0.7.0"
+DOWNLOAD_FILE="${VERSION}.${OS}.x86_64.tar.xz"
+wget https://storage.googleapis.com/shellcheck/"${DOWNLOAD_FILE}"
```

Comment:
hah, my system didn't have wget. Isn't bash amazing.

Can you add a failure case if wget doesn't exist on the system with exit 1 saying `wget` is missing?

Reply:
Let's use curl, please 😄

Reply:
that is also acceptable :D

Reply:
done. ptal